src/client/lib/utils.js

```diff
@@ -7,7 +7,7 @@ var doc = document;
  * @params {Object} obj
  * @return {String} A string representation of the Object passed in.
  */
-export var serialize = (obj) => JSON.stringify(obj || {});
+export let serialize = (obj) => JSON.stringify(obj || {});
```

Comment:
'export' is only available in ES6 (use esnext option).<br>'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).<br>'arrow function syntax (=>)' is only available in ES6 (use esnext option).